Update Adyen Doc (#5993)
vuestorefront/vue-storefront: packages/core/docs/integrations/adyen.md

Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).

## FAQ

> ### How to debug?
> Open browser's `Network` tab. Each request related to the payment will have Commercetools Payment object in the response. You could look at `custom.fields.SOME_FIELD` to discover what has been sent to the Adyen and what was the response (or error!).

Reviewer: ```suggestion

### How to debug data flow?

Open the` Network` tab in the browser's devtools. Each payment request will have commercetools [Payment object](https://docs.commercetools.com/api/projects/payments#payment) in the response. You can check `custom.fields.SOME_FIELD` to see what has data was sent to Adyen and what was the response (or error).

```

Reviewer: Please specify what `custom.fields.SOME_FIELD` is. This name is not used anywhere else in this document.

### Klarna Pay Later does not work for United States
Klarna Pay Later is not supported in the United States. However, sometimes it is added when you enable Klarna in Adyen's dashboard. If you have this problem, contact Adyen's support to remove it.

> ### Nothing happens after 3DS2 Auth in one environment but works in a different environment
> There might be a situation when you can finish 3DS2 Auth in the local environment. But it is impossible on staging. It simply means you've provided `http://localhost:3000` as an `origin` in the `middleware.config.js` instead of your staging's URL. 
> 
> It can work properly only on provided `origin`.

Reviewer: ```suggestion
### 3DS2 Auth doesn't work in one environment

There might be a situation when you can finish 3DS2 Auth in the local environment but not in the other, like staging. When this happens, make sure to change `origin` in the `middleware.config.js` from `http://localhost:3000` to the URL of your staging environment. 
```

### Structure of DetailsRequest contains the following unknown fields...

> Update version of the both extension module and notification module. Just modify tag to the newest version in `extension.Dockerfile` and `notification.Dockerfile`.

Reviewer: ```suggestion

Update extension and notification modules to the newest version by updating the tag in `extension.Dockerfile` and `notification.Dockerfile`.

```

Reviewer: I suggest being more precise with the work `newest` since it might be different in the future. If you think it's okay, we can leave it.
